### DISCUSSION TITLE
Dev Container for F prime

### DISCUSSION THREAD

**Discussion by Sankie005** [nasa/fprime#2976] [Ideas]
Hi! This is relating to the recent in person workshop for Fprime@JPL where someone made a request for having a premade dev container for F prime, I'd be happy to implement something like this for Vscode but I'd like to gauge the enthusiasm and need for something like this before I commit to developing it.

**Comment by thomas-bc** [comment]
Linking relevant discussons:

https://github.com/nasa/fprime/issues/1806
https://github.com/nasa/fprime/discussions/908

Also https://github.com/fprime-community/fprime-workspace-image

cc @capsulecorplab

**Reply by capsulecorplab** [reply]
Yup, I put the fprime-workspace-image together so that fprime developers have an immutable dev environment that "just works" out of the box. I haven't gotten fprime v3.5.0 to work, but the latest release (which uses v3.4.0 at the time of writing) should build the baremetal reference as well as all the tutorials.
@Sankie005 feel free to open an issue if you encounter any problems.

**Reply by Sankie005** [reply]
Okay thanks for your work and response! Will try it out and let you know
